File: src/commonMain/kotlin/spaceEngineers/model/SeBlockDefinition.kt
package spaceEngineers.model


data class SeBlockDefinition(
    val id: String,
    val blockType: String,
    val buildProgressModels: List<SeBuildProgressModel>,
    val size: Vec3,
    val cubeSize: CubeSize,
    val mountPoints: List<MountPoint>,
    val public: Boolean,
    val availableInSurvival: Boolean,
    val enabled: Boolean,
)

File: Source/Ivxr.SePlugin/WorldModel/SeBlockDefinition.cs
using System.Collections.Generic;
using VRageMath;

namespace Iv4xr.SePlugin.WorldModel
{
    public class SeBlockDefinition
    {
        public string Id;
        public string BlockType;
        public List<SeBuildProgressModel> BuildProgressModels;
        //public string Model;
        public string CubeSize;
        public Vector3I Size;
        public List<SeMountPoint> MountPoints;
        public bool Public;
        public bool AvailableInSurvival;
        public bool Enabled;
    }
}
